lab/environments.py

```diff
@@ -132,14 +132,13 @@ class SlurmEnvironment(Environment):
     specify a project account (needed on NSC if you're part of multiple
     projects). ::
 
-        extra_options="#SBATCH --nodelist=ase[1-5,7,10]"
+        extra_options="#SBATCH --nodelist=ica[1-5,7,10]"
         extra_options="#SBATCH --account=naiss2024-5-421"
 
 
     *partition* must be a valid Slurm partition name. In Basel you
     can choose from
 
-    * "infai_1": 24 nodes with 16 cores, 64GB memory, 500GB Sata (default)
     * "infai_2": 24 nodes with 20 cores, 128GB memory, 240GB SSD
     * "infai_3": 12 nodes with 128 cores, 512GB memory, 240GB SSD
 
```

Comment:
Can you please grep for "infai_1" and remove the remaining occurrences as well?

Reply:
I removed two more cases and left the one in `news.rst`. Did you want to change the example experiment? It's not tested on the new partition, even though I see no problem there.